### ISSUE TITLE
Heroku crashes

### ISSUE THREAD

**Issue by bovas85** [spences10/twitter-bot-bootstrap#23]
Hi there, amazing work and I hope to contribute.

I'm getting the following error with heroku when deployed after a few loops:
```
Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch
Stopping process with SIGKILL
Process exited with status 137
State changed from starting to crashed
```

I've followed your guide but can't seem to find the issue

**Comment by bovas85** [comment]
found a workaround using express
var express = require('express');
var app     = express();

app.set('port', (process.env.PORT || 5000));

//For avoidong Heroku $PORT error
app.get('/', function(request, response) {
    var result = 'App is running'
    response.send(result);
}).listen(app.get('port'), function() {
    console.log('App is running, server is listening on port ', app.get('port'));
});

**Comment by spences10** [comment]
Hey @bovas85  thanks for taking the time to log an issue. I have seen the issue previously and (I thought) clearly documented it in the README. This particular issue was quite tricky, but once you play around with the worker settings it usually sorts itself out.

```
$ heroku ps:scale worker=0
$ heroku ps:scale worker=1
```

If that didn't work the I'd play with the dashboard settings in Heroku.

I'll take a look at your solution later I'm on mobile currently, ease feel free to make a PR

**Comment by bovas85** [comment]
it now gives me another issue:

```
2017-02-12T15:52:47.356888+00:00 heroku[web.1]: Idling
2017-02-12T15:52:47.357378+00:00 heroku[web.1]: State changed from up to down
2017-02-12T15:52:48.278071+00:00 heroku[web.1]: Stopping all processes with SIGTERM
2017-02-12T15:52:48.438833+00:00 heroku[web.1]: Process exited with status 143
```

**Comment by bovas85** [comment]
It appears I had to add index.html to the root folder to prevent the app from timing out on the app.get request.
I'll reply if there are more issues, otherwise I'll close the issue and create a PR

**Comment by spences10** [comment]
@bovas85 ok, apologies if it's causing you pain, it was something that caused me a fair bit of searching to resolve. 

And I do encounter the issue time to time now but usually setting the workers from the CLI fixes it, manually toggling the workers on the Heroku dashboard if that doesn't do anything.

**Comment by bovas85** [comment]
I'm still getting that error... How do I disable it from the dashboard?

**Comment by spences10** [comment]
Hey @bovas85 I have updated the readme with further details on Heroku

I hope this explains it for you, this is as much as I can do for you my friend :+1: 

Resolved with #29
